### PR TITLE
Upgrade jacoco to 0.8.3

### PR DESCRIPTION
**Pull request by iirina**

### CODE REVIEW

File: src/test/shell/bazel/bazel_coverage_java_test.sh [hunk 1]
```diff
@@ -21,6 +21,174 @@ CURRENT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
...
+    cat >>WORKSPACE <<EOF
+http_archive(
+    name = "local_java_tools",
+    urls = ["${JAVA_TOOLS_ZIP_FILE_URL}"]
+)
+EOF
+    fi
+
+    cat >>WORKSPACE <<EOF
+http_archive(
```
Comment (lberki) [inline src/test/shell/bazel/bazel_coverage_java_test.sh]: Why these changes?

**Comment by iirina** [comment]
I rebased the PR and many reviewers were automatically added by GH. Sorry everyone! PR is still WIP.

**Comment by iirina** [comment]
Abandoned PR
